checker/tests/nullness/ReplacePolyNull.java

```diff
@@ -0,0 +1,19 @@
+import org.checkerframework.checker.nullness.qual.PolyNull;
+
+public class ReplacePolyNull {
+
+    @PolyNull String foo(@PolyNull String param) {
+        if (param != null) {
+            //  @PolyNull is really @Nullable, so change
+            // the type of param to @Nullable.
+            return param;
```

Comment:
```suggestion
            return param.toString();
```

To illustrate that `param` is `@NonNull` here.

Reply:
Thanks for the advise, done.